doc/Makefile

```diff
@@ -53,6 +53,7 @@ clean:
 
 html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+	jsdoc ../emperor/support_files/js/ -d $(BUILDDIR)/jsdoc -c ../jsdoc-config.json -r
```

Comment:
Correct, the file is added in #477 which this depends on. I can move it over in that PR if you want.